platform/darwin/test/MGLFunctionTests.m

```diff
@@ -0,0 +1,13 @@
+#import <XCTest/XCTest.h>
+#import <Mapbox/Mapbox.h>
+
+@interface MGLFunctionTests : XCTestCase
```

Comment:
Ah, I see. Alternatively, maybe we can call this class MGLStyleValueTests and turn the Swift class into an extension on MGLStyleValueTests. Not sure if XCTest will be happy with that or not.